code/datums/components/milkable.dm

```diff
@@ -0,0 +1,67 @@
+// how cows and other simplemobs are able to be milked
+
+/datum/component/milkable
+	var/mob/producer
+	var/obj/item/reagent_containers/intstorage
+	var/max_reagents
+	var/min_reagents
+	var/datum/reagent/reagent_generated
+	var/datum/callback/on_milked
+	var/list/allowed_containers = list(/obj/item/reagent_containers/glass)
+
+/datum/component/milkable/Initialize(_max_reagents, _starting_amount, _reagent_used, _callback_milked, _allowed_containers)
+	if(!ismob(parent))
```

Comment:
Add !ishuman check to this

Reply:
arent humans mobs?

Reply:
This is the point
I don't want to see milkable on humans

Reply:
idk..if a senior admin wants to do something with your code it would be rude to make it impossible right

Reply:
i added like 3 different "yes im sure" checks to the addcomponent milkable, should be good now